Samples/azure-storage/Azure.Ruby/azure_storage/models/usage_list_result.rb

```diff
@@ -12,9 +12,6 @@ class UsageListResult
       # @return [Array<Usage>] Gets or sets the list Storage Resource Usages.
       attr_accessor :value
 
-      # @return [String]
```

Comment:
looks like commit [64a7b867315c](https://github.com/Azure/autorest/commit/64a7b867315c5e6b23222ca86dab5ca3c7d2580e) has changed the samples but I don't see any changes in Ruby samples with that check-ins.